tests/test_data/cube_response_filtered_stratified.json

```diff
@@ -14,44 +14,44 @@
             "stratifier": "",
             "rows": [
                 [
-                    16917,
-                    "female"
```

Comment:
this was always supposed to be a label/number ordered pairing - this ordering was a product of the old assembly method and our test data. so the new method is handling an unexpected potential bug case (great!). but that's the reason for the in-array ordering change.

The actual order of the rows themselves doesn't matter, so not worrying too much about the shift in order of the individual rows in some cases.